.muse/config.toml

```diff
@@ -0,0 +1 @@
+jdk11 = true
```

Comment:
Could you add a comment or two pointing to the documentation how to add/change/skip checks?

Reply:
@tflobbe Sure, I'll put that here and amend the commit.

The full documentation is docs.muse.dev.  Most interesting is the repository configuration reference https://docs.muse.dev/docs/repository-configuration/ which details how to change which tools runs, add custom tools (other than those built into the platform by default), filter for or against certain bug types, etc.

For example, an explicit `.muse/config.toml` file might be:

```
jdk11 = true
build = "./gradlew assemble"
tools = [ "infer", "errorprone", "findsecbugs" ]
customTools = [ ]
```

Reply:
Ah, I noticed you added the comments on the commit. Thanks for that, but I was thinking we could put some comment in the file itself, something like:
```
# This file can configure Muse PR static analysis. 
# Docs for adding, removing or changing checks can be found here: https://docs.muse.dev/docs/repository-configuration/
```
or something of that sort, mostly to save some googling. WDYT?
@dsmiley, do you have any concerns with this? (since you commented on the Jira issue). I seems like a great thing to try, and if Apache infra installed the app for us, I see no reason not to do it.

Reply:
I've added the comment to the file as well now.